modules/oe_content_entity/modules/oe_content_entity_organisation/config/install/core.entity_form_display.oe_organisation.oe_stakeholder.default.yml

```diff
@@ -0,0 +1,92 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.oe_organisation.oe_stakeholder.field_oe_acronym
+    - field.field.oe_organisation.oe_stakeholder.field_oe_address
+    - field.field.oe_organisation.oe_stakeholder.field_oe_contact_url
+    - field.field.oe_organisation.oe_stakeholder.field_oe_logo
+    - field.field.oe_organisation.oe_stakeholder.field_oe_website
```

Comment:
We are not using field_ as prefix for field names.

Reply:
Changed.